crash-reporting-napier/src/commonMain/kotlin/dev/icerock/moko/crashreporting/napier/CrashReportingAntilog.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.crashreporting.napier
+
+import com.github.aakira.napier.Antilog
+import com.github.aakira.napier.Napier
+import dev.icerock.moko.crashreporting.core.CrashReportingCore
+import dev.icerock.moko.crashreporting.core.ExceptionLogger
+
+class CrashReportingAntilog(private val exceptionLogger: ExceptionLogger) : Antilog() {
+    override fun performLog(
+        priority: Napier.Level,
+        tag: String?,
+        throwable: Throwable?,
+        message: String?
+    ) {
+
+        exceptionLogger.log(
+            CrashReportingCore().getLogText(
```

Comment:
every time do object instantiation is not good. change `CrashReportingCore` to `object` (singleton)